test/packer_oci_example_test.go

```diff
@@ -0,0 +1,43 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/oci"
+	"github.com/gruntwork-io/terratest/modules/packer"
+)
+
+const compartmentOcid = "ocid1.compartment.oc1..aaaaaaaa"
```

Comment:
What is this?

Reply:
[Compartment](https://docs.cloud.oracle.com/iaas/Content/GSG/Tasks/choosingcompartments.htm) is a logical group of OCI resources and is basically a mandatory parameter for any call to OCI. It's one of the [mandatory parameters](https://www.packer.io/docs/builders/oracle-oci.html#required) of OCI _Packer_ builder as well.

I extracted it out of a test because it will be probably shared across multiple tests. But probably would be better to extract it from the source code completely. I will research this further.